### DISCUSSION TITLE
Not listed in Add Integrations?

### DISCUSSION THREAD

**Discussion by anthonyallure** [briis/unifiprotect#262] [Q&A]
I'm pretty new to Home Assistant and still getting my head around it... I've read the documentation about adding Unifi Protect to HA and followed instructions but when I come to add it using the integrations page it's not listed and I'm not sure how you go about installing manually?

Any ideas how to get it to show in the add integrations page? I've restarted server, tried multiple browsers, etc but still doesn't show up.

**Comment by briis** [comment]
Let me give you a short crash-course 😁

Home Assistant comes with a lot of built-in Integrations, that are always available from the Integrations Page, these I call Core Integrations.

But an Integration like Unifi Protect, is not a Core Integration, so before it will show up on the Integrations page, you need to add the necessary files to Home Assistant. 
You can do this be either copying the files manually or use HACS, which is a store for Non Core Integrations like Unifi Protect (You can find a link in the README file). 

I really recommend spending the time to install HACS, as it will be a lot easier going forward to install updates and other integrations.

So once you select Unifi Protect in HACS, and install it, the only thing that happens is that the necessary files are copied to Home Assistant. Then you need to re-start Home Assistant, for the system to pick-up the new files. At this point, Unifi protect is now part of Integrations that can be installed from the Integrations page.

But, this page often gets cached in your browser, so to ensure you can find Unifi Protect, go to the Integrations page, and do a hard refresh of your browser (CMD+R on a Mac). 
Now press the button in the lower right corner named *+ ADD INTEGRATION*, and search for Unifi Protect. It should now be available to configure.

Hope this helps

**Reply by anthonyallure** [reply]
That's made it much clearer for me! Thanks for taking the time to explain that - I'll give it ago! 👍
